README.md

```diff
@@ -10,6 +10,7 @@ To begin using this template, choose one of the following options to get started
 * [Download the latest release on Start Bootstrap](http://startbootstrap.com/template-overviews/freelancer/)
 * Clone the repo: `git clone https://github.com/BlackrockDigital/startbootstrap-freelancer.git`
 * Fork the repo
+* Use command (when you are in appropriate directory): npm install
```

Comment:
use markdown syntax for commands: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code